components/organisms/o-microcart.vue

```diff
@@ -11,8 +11,8 @@
             :key="product.id"
             :image="getThumbnailForProductExtend(product)"
             :title="product.name"
-            :regular-price="getProductRegularPrice(product)"
-            :special-price="getProductSpecialPrice(product)"
+            :regular-price="getProductPrice(product.totals).regular"
```

Comment:
I didn't know that, thanks for finding out that case  👍 
Now I've introduced new helper function `getProductPriceFromTotals()` which is used in microcart in online mode and if `product.totals` exists. Otherwise `getProductPrice()` is used.